index.html

```diff
@@ -22,15 +22,16 @@
   <!-- Header Section Start -->
   <header>
     <nav class="wrapper padding-inline-300">
-      <div class="cub">
+      <div class="area">
+        <div class="cub"></div>
         <a href="#" class="logo">Mahdi Rezaei</a>
       </div>
       <ul class="main-nav">
         <li><a href="#portfolio">Portfolio</a></li>
         <li><a href="#about">About</a></li>
         <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Kindly make sure nav links takes you to their respective section. You can fix this by adding `id` to each section. 😄